docs/detections/detection-engine-intro.asciidoc

```diff
@@ -86,7 +86,9 @@ Indicator match rules provide a powerful capability to search your security data
 
 In addition, the following support restrictions are in place:
 
-* {elastic-sec} does not support the use of either cold or frozen {ref}/data-tiers.html[tier data] with indicator match rules.
+* Indicator match rules don't support cold or frozen data, but will query cold and frozen {ref}/data-tiers.html[data tiers] if they exist. To exclude query results from cold and frozen tiers, configure the `excludedDataTiersForRuleExecution` <<exclude-cold-frozen-data-rule-executions,advanced setting>> (which applies to all rules in a space), or add a <<exclude-cold-frozen-data-individual-rules,Query DSL filter>> to individual rules. 
```

Comment:
`if they exist` -> `if they match the rule's index pattern` ?

Reply:
I think this should be updated to remove the reference to the "advanced setting" as it's not helpful in the context of IM rules. For IM rules, what helps the user is the workflow for excluding tiers for **SINGLE** rules, not for **ALL** rules.

Reply:
```suggestion
* Indicator match rules don't support cold or frozen data but will query cold and frozen {ref}/data-tiers.html[data tiers] if they match the rule's index pattern. To exclude query results from cold and frozen tiers, create a <<exclude-cold-frozen-data-individual-rules,Query DSL filter>> and add it to individual rules. 
```